### ISSUE TITLE
Training loss is nan?

### ISSUE THREAD

**Issue by zhaoyinjiang9825** [yeyupiaoling/PPASR#3]
[2021-07-14 02:09:40.075967] Train epoch: 0, batch: 750/34799, loss: 6.70559, learning rate: 0.0001, train time: 4.319s
完成第750的保存
[2021-07-14 02:10:20.735872] Train epoch: 0, batch: 760/34799, loss: 6.64842, learning rate: 0.0001, train time: 4.218s
完成第760的保存
[2021-07-14 02:11:05.361007] Train epoch: 0, batch: 770/34799, loss: 6.76281, learning rate: 0.0001, train time: 6.372s
完成第770的保存
[2021-07-14 02:11:53.009762] Train epoch: 0, batch: 780/34799, loss: 6.61592, learning rate: 0.0001, train time: 3.155s
完成第780的保存
[2021-07-14 02:12:37.086634] Train epoch: 0, batch: 790/34799, loss: nan, learning rate: 0.0001, train time: 3.768s
完成第790的保存
[2021-07-14 02:13:21.365857] Train epoch: 0, batch: 800/34799, loss: nan, learning rate: 0.0001, train time: 3.402s
完成第800的保存
[2021-07-14 02:14:00.991322] Train epoch: 0, batch: 810/34799, loss: nan, learning rate: 0.0001, train time: 3.514s
完成第810的保存
[2021-07-14 02:14:50.522749] Train epoch: 0, batch: 820/34799, loss: nan, learning rate: 0.0001, train time: 3.567s
完成第820的保存
[2021-07-14 02:15:37.339313] Train epoch: 0, batch: 830/34799, loss: nan, learning rate: 0.0001, train time: 3.885s
How to deal with it？

**Comment by yeyupiaoling** [comment]
@zhaoyinjiang9825 你有没有改动什么？

进阶版本我还没有完整训练，你先用入门级先

**Comment by zhaoyinjiang9825** [comment]
我只换了训练数据，别的都没有改动
